pkg/env/costmodelenv.go

```diff
@@ -67,6 +67,7 @@ const (
 	ETLEnabledEnvVar                     = "ETL_ENABLED"
 	ETLMaxPrometheusQueryDurationMinutes = "ETL_MAX_PROMETHEUS_QUERY_DURATION_MINUTES"
 	ETLResolutionSeconds                 = "ETL_RESOLUTION_SECONDS"
+	ETLStoreReadOnlyMode                 = "ETL_STORE_READ_ONLY"
```

Comment:
Agree with Bolt's comment that it's rough already having `ETL_READ_ONLY`... but this seems like making the best of a messy situation.